MobileCenter/MobileCenterTests/MSChannelDefaultTests.m

```diff
@@ -328,10 +322,10 @@ - (void)testDeleteDataOnDisabled {
   [self initChannelEndJobExpectation];
   id senderMock = OCMProtocolMock(@protocol(MSSender));
   id storageMock = OCMProtocolMock(@protocol(MSStorage));
-  id<MSLog> log = [MSAbstractLog new];
+  id logMock = [self getValidMockLog];
```

Comment:
Somehow `mockLog` sounds better to me, but I that's probably my own taste in variable names ;)